ui/lib/core/addon/helpers/has-capability.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/**
+ * helper to get capabilities from map for given item by id
+ * useful in list views when capabilities are fetched for each item
+ * provide the full map of capabilities along with the id of the item and capabilities to be evaluated (read, update etc)
```

Comment:
is `id` the dynamic part of the path used to request the capability?

Reply:
Yes that's right. Originally I thought about passing in the full path or the key to the path map and dynamic segment but I settled on this for now. I could see needing to iterate on this if more complex usage is needed.